contrib/fuzz/diff_fuzz_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 ADA Logics Ltd
```

Comment:
I would say the copyright for all the fuzz tests should be containerd's. As the author of these fuzz tests and on behalf of Ada Logics, I approve reassigning copyright.

Reply:
updated